helix-lsp/src/lib.rs

```diff
@@ -888,18 +892,70 @@ fn start_client(
     doc_path: Option<&std::path::PathBuf>,
     root_dirs: &[PathBuf],
     enable_snippets: bool,
-) -> Result<NewClient> {
+) -> Result<Option<NewClient>> {
+    let (workspace, workspace_is_cwd) = helix_loader::find_workspace();
+    let workspace = path::normalize(&workspace);
```

Comment:
```suggestion
    let workspace = path::normalize(workspace);
```

I'm surprised that clippy doesn't catch this: `workspace` here already implements `AsRef<Path>` so the borrow is needless